hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/handlers/BucketEntityHandler.java

```diff
@@ -142,7 +145,15 @@ public DUResponse getDuResponse(
     }
     duResponse.setCount(dirDUData.size());
     duResponse.setSize(bucketDataSize);
+
+    if (sortSubpaths) {
+      // Parallel sort directory/files DU data in descending order of size and returns the top N elements.
+      dirDUData = sortDiskUsageDescendingWithLimit(dirDUData,
+          DISK_USAGE_TOP_RECORDS_LIMIT);
```

Comment:
Can we make this limit pass as parameter instead of hardcoding here ? Just move one level above.

Reply:
As discussed, we will ignore this comment for this PR and address it in another one.